CHANGELOG.rst

```diff
@@ -17,6 +17,14 @@ This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 .. towncrier release notes start
 
+[1.9.3] - 2020-03-27
+^^^^^^^^^^^^^^^^^^^^
+
+Bugfixes
+--------
+- `#5505 <https://github.com/rasahq/rasa/issues/5505>`_: Set default value for ``weight_sparsity`` in ``ResponseSelector`` to ``0``.
```

Comment:
Can we make this clearer that it fixes the response selector, so that poeple know that if they're using it they should be on this version?

Reply:
@erohmensing Makes sense, this was added in https://github.com/RasaHQ/rasa/pull/5505. Can I make it clearer by changing the `CHANGELOG.rst` directly in this branch itself?

Reply:
Added some explanation below.

Reply:
@dakshvar22 maybe if it is used in their pipeline? or if they have responses? 

just bc the default pipeline has included it since 1.8 regardless of if the user is using it

Reply:
Updated